tests/model_fields/test_filefield.py

```diff
@@ -83,3 +84,10 @@ def test_move_temporary_file(self):
             tmp_file_path = tmp_file.temporary_file_path()
             Document.objects.create(myfile=tmp_file)
             self.assertFalse(os.path.exists(tmp_file_path), 'Temporary file still exists')
+
+    def test_open_returns_self(self):
+        d = Document.objects.create(myfile='something.txt')
+        # replace the FileField's file with an in-memory ContentFile,
```

Comment:
"Replace" (captialize)
chop comma since "so that ..." isn't an independent clause (couldn't be a sentence on its own).
Add period at the end of the sentence.